addons/binding/org.openhab.binding.tankerkoenig/README.md

```diff
@@ -110,16 +110,56 @@ java.io.IOException: java.util.concurrent.ExecutionException: javax.net.ssl.SSLH
 ......
 ```
 
-That indicates a missing certificate for the https-connection on the system.
-In order to get the required certificate on a Linux-system one needs to perform these steps:
+That indicates a missing certificate of a certification authority (CA) in the certificate-store of the java jdk under which openHAB is running.
+In most cases an update to the latest jdk version does solve this, because the store of the cacerts are maintained with the java versions.
```

Comment:
Consider, "In most cases, updating to the latest version of jdk solves this because the store of cacerts are maintained and updated in java releases."